doc/gh-pages/pages/development/development.md

```diff
@@ -1,10 +1,39 @@
 ---
-title: Develop
+title: Development
 has_children: true
-nav_order: 5
+nav_order: 4
 ---
-# Develop
+# How to contribute to the MNE-CPP Project
+
+## Welcome aboard!
+
+Before you start, there is a few basic concepts that should be understood. We use a variety of different tools to develop and maintain MNE-CPP.
+
+## Before you start
+
+Learn C++ and the [Qt](https://www.qt.io/){:target="_blank" rel="noopener"}. 
+
+We mainly use C++ programming language and Qt, for development purposes. We are very happy with the choice, but these technologies can be quite tricky. There are plenty of fantastic resources out there to be able to learn both, but if you feel stuck, go ahead and ask us. Since you intend to become a contributor, we recommend you to start using Github and open an issue in the MNE-CPP project, in order to communicate with the development team.
```

Comment:
Do we want the point of contact with us to be through opening an issue? maybe this is a good time to implement the 'github discussions' feature @LorenzE has been mentioning.

Reply:
Agreed. I agree a lot. 
Let's do that.